gnss_initpose_node:
    map_frame: "map"
    base_frame: "base_footprint"
    publish_tf: false
    gnss_topic: "fix"
    point_topic: "initialpose"
    latlong_param: "+proj=latlong +ellps=WGS84"
    coord_param: "+init=epsg:2451"
    map_lat: 35.6897 #deg
    map_long: 140.02138 #deg
    map_frame_direction: 0 #deg